pkgs/build-support/build-fhsenv-bubblewrap/buildFHSEnv.nix

```diff
@@ -15,6 +15,10 @@
 , extraBuildCommands ? ""
 , extraBuildCommandsMulti ? ""
 , extraOutputsToInstall ? []
+# Whether to symlink /usr/lib* or copy files directly
+# This is mainly a workaround for steam pressure-vessel bug tracked at
```

Comment:
Giga nit: Steam, capitalized. Also, might be worth reporting at https://gitlab.steamos.cloud/steamrt/steam-runtime-tools/-/issues, which is a bit more targeted?

Reply:
I've read somewhere in their docs to better report issues on the GitHub. Probably that they can manage things more easily and they had similar issues in that repo in the past.